### PR TITLE
chore: rm chokidar-cli & add nodemon@latest

### PR DESCRIPTION
**Pull request by DobroTora**
Replaces `chokadir` with `nodemon` due to security vulnerability and updates `dev:docs` script to use `nodemon` to watch for changes in target directories. Moves `semantic-release` and `nodemon` to `devDependencies`

### CODE REVIEW

File: package.json
Comment (edleeks87) [inline package.json:64]: you can remove this line

Comment (DobroTora) [reply package.json]: Done

File: package.json
Comment (edleeks87) [inline package.json:26]: This doesn't look like the correct syntax for using `nodemon`, have you confirmed it's correct and works? The name of the script (`dev:docs`) also suggests it's run locally, if that's the case then it might be possible to move to a devDependency.

https://www.npmjs.com/package/nodemon

**Review by edleeks87 — CHANGES_REQUESTED** [review]
Before this PR is merged it should be squashed to a single commit, the commit message will need changing to something more reflective of the change: `Yargs` doesn't seem to be a dependency here let alone being updated to `16.20`. You are replacing `chokidar-cli` with `nodemon` so the commit message should reflect that

**Review by edleeks87 — COMMENTED** [review]
@DobroTora as best I can tell this works, it does execute the script (to update the json) when you run the `dev:docs` which it doesn't when running it in master. I haven't been able to find a way to stop this, but I don't think this is an issue as it does watch and update the file when any changes are made in the given directories. I will be happy to approve when you remove the extra blank line from the `package.json` file (please do it as an amend commit). I think it would be good to get @jamime or @nicktitchmarsh to review/ approve as they may have more context that I've missed

**Review by edleeks87 — APPROVED** [review]

File: package.json
Comment (jamime) [inline package.json:64]: This should be a dev dependency

**Review by jamime — CHANGES_REQUESTED** [review]

**Comment by edleeks87** [comment]
Looks good @DobroTora, you might as well squash the commits as one commit will suffice for this update

**Review by edleeks87 — APPROVED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

**Pull request merged**
